website/server/models/group.js

```diff
@@ -90,6 +90,10 @@ schema.plugin(baseModel, {
   noSet: ['_id', 'balance', 'quest', 'memberCount', 'chat', 'challengeCount'],
 });
 
+schema.virtual('quest.participatingMembers').get(function getParticipatingQuestMembers () {
```

Comment:
can we transform this in a method? I find virtual getters and setters confusing, with a method you immediately now that you're computing a result